### DISCUSSION TITLE
Is it possible to show child tab count while expanded

### DISCUSSION THREAD

**Discussion by Vindickator** [piroor/treestyletab#3135] [Q&A] [answered]
Hello,

By default the child tab count is shown while the tabs are collapsed, 
is it possible to have the count shown even during the tree expanded?

**Existing: Tree collapsed shows count**
![image](https://user-images.githubusercontent.com/87226625/170819891-ec8bfa43-773e-4d6a-a123-81f3448a4020.png)

**Existing: Tree expanded doesn't show count**
![image](https://user-images.githubusercontent.com/87226625/170819908-7efae990-357c-4a21-aca9-5079d0f823c4.png)

**Expectation: Tree expanded shows count**
![image](https://user-images.githubusercontent.com/87226625/170819884-a73dd140-8469-41cb-899e-e6b236044efe.png)

Thanks in advance.
Vin

**Comment by piroor** [comment, marked as answer]
You need to cancel this declaration: https://github.com/piroor/treestyletab/blob/db88d8be8958eda1c2f494de2aeb9300f26dd6aa/webextensions/sidebar/styles/base.css#L629-L632

For example:

```css
tab-item[data-child-ids]:not(.subtree-collapsed) .counter {
  display: initial !important;
}
```

**Comment by Vindickator** [comment]
Thanks
